zh/deploy-on-gcp-gke.md

```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
     Outputs:
 
-    how_to_connect_to_default_cluster_tidb_from_bastion = mysql -h 172.31.252.20 -P 4000 -u root
     how_to_ssh_to_bastion = gcloud compute ssh tidb-cluster-bastion --zone us-west1-b
-    how_to_set_reclaim_policy_of_pv_for_default_tidb_cluster_to_delete = kubectl --kubeconfig /.../credentials/kubeconfig_tidb-cluster get pvc -n tidb-cluster -o jsonpath='{.items[*].spec.volumeName}'|fmt -1 | xargs -I {} kubectl --kubeconfig /.../credentials/kubeconfig_tidb-cluster patch pv {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
     kubeconfig_file = ./credentials/kubeconfig_tidb-cluster
-    monitor_lb_ip = 35.227.134.146
-    monitor_port = 3000
-    region = us-west1
-    tidb_version = v3.0.1
     ```
 
+## 部署 TiDB 集群和监控
+
+1. 准备 TidbCluster 和 TidbMonitor CR 文件：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+	  cp manifests/{db,db-monitor}.yaml .
```

Comment:
The files have suffix `example` in the repo.

Reply:
fixed